spec/components/find_interface/header/view_preview.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module FindInterface::Header
+  class ViewPreview < ViewComponent::Preview
+    def with_custom_service_name
+      render(Header::View.new(service_name: "Hello"))
+    end
+
+    def with_our_service_name
+      render(Header::View.new(service_name: I18n.t("service_name.find")))
```

Comment:
Same thing?

https://publish-teacher-training-pr-2922.london.cloudapps.digital/view_components/header/view/with_our_service_name

Reply:
Added for consistency since we have duplicated.